tests/sdk/go/go_test.go

```diff
@@ -26,7 +26,7 @@ import (
 var baseOptions = &integration.ProgramTestOptions{
 	Verbose: true,
 	Dependencies: []string{
-		"github.com/pulumi/pulumi-kubernetes/sdk/v3/go",
+		"github.com/pulumi/pulumi-kubernetes/sdk/v3",
```

Comment:
I don't think this is right. Pretty sure I had to change this recently to get tests passing.

Reply:
I was running into issues getting the test to run locally without this since the test framework installs a replace block mapping to `pulumi/pulumi-kubernetes/sdk/go` which doesn't have a go.mod. It did work locally with the above change but the CI run is failing. Will investigate.